rosserial_python/src/rosserial_python/SerialClient.py

```diff
@@ -545,7 +538,7 @@ def run(self):
                 else:
                     rospy.loginfo("wrong checksum for topic id and msg")
 
-            except IOError as exc:
+            except ImportError as exc:
```

Comment:
What could possibly raise an `ImportError` that is caught here?

Reply:
I agree, this makes no sense. Reverting.